qiskit/circuit/library/standard_gates/siswap.py

```diff
@@ -0,0 +1,77 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""sqrt(iSWAP) gate."""
+
+import numpy as np
+from qiskit.circuit.gate import Gate
+
+
+class SiSwapGate(Gate):
+    r"""sqrt(iSWAP) gate.
+
+    A 2-qubit symmetric gate from the iSWAP (or XY) family.
+    It has Weyl chamber coordinates (π/8, π/8, 0).
+
+    Can be applied to a :class:`~qiskit.circuit.QuantumCircuit`
+    with the :meth:`~qiskit.circuit.QuantumCircuit.SiSwap` method.
+
+    .. parsed-literal::
+
+                 ┌────────────┐┌────────────┐
+            q_0: ┤0           ├┤0           ├
+                 │  Rxx(-π/4) ││  Ryy(-π/4) │
+            q_1: ┤1           ├┤1           ├
+                 └────────────┘└────────────┘
+
+    .. math::
+        B\ q_0, q_1 =
```

Comment:
Should this be 
```suggestion
        \sqrt{\mathrm{iSwap}}\ q_0, q_1 =
```
?